### PR TITLE
Style changes

### PR DESCRIPTION
**Pull request by RichardJJG**
Fixes #3876

### CODE REVIEW

**Comment by RichardJJG** [comment]
> @RichardJJG did you find out whether the correct way to update the configmap in the second bullet of step 1 is using `kubectl -n <namespace> edit configmap <resource-name>`

I've reached out in Slack for that: https://knative.slack.com/archives/C93E33SN8/p1630401404010400

File: docs/developer/serving/services/custom-domains.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ serve a Knative Service at this domain.
 
 !!! note
     If you create a domain mapping to map to a [private Knative Service](private-services.md),
-    the private Knative Service will be accessible from public internet with the custom domain of the domain mapping.
+    the private Knative Service is accessible from public internet with the custom domain of the domain mapping.
```
Comment (abrennan89) [inline docs/developer/serving/services/custom-domains.md:14]: I'd use DomainMapping just to keep it consistent with the rest of the doc

**Review by abrennan89 — CHANGES_REQUESTED** [review]
Minor suggestion

**Comment by RichardJJG** [comment]
> Minor suggestion

Done, thanks, @abrennan89!

**Comment by abrennan89** [comment]
/lgtm
/approve

@RichardJJG there's a hold label on this, not sure if you want me to remove and merge but lmk if you need anything else, you should be able to remove that label yourself 🙂

**Comment by RichardJJG** [comment]
> /lgtm
> /approve
> 
> @RichardJJG there's a hold label on this, not sure if you want me to remove and merge but lmk if you need anything else, you should be able to remove that label yourself 🙂

@abrennan89, thanks, I forgot!

**Pull request merged**